UC-Berkeley-Extension/global
# Drupal install

This PR builds on work in #1 and installs the Drupal site.

See the instructions in the README for this branch.

To test:

-- Follow the `New Installations` instructions.
-- After install, try changing a configuration, then run `drush cex`
-- Run `git status` and confirm a config change in `/config/drupal`
-- Run `git stash` to remove your config change.
-- Run `drush cim` to reset config.
